runtime/parser2/expression.go

```diff
@@ -626,6 +615,95 @@ func defineLessThanOrTypeArgumentsExpression() {
 		})
 }
 
+func defineGreaterThanOrBitwiseRightShiftExpression() {
```

Comment:
```suggestion
// parse "greater than" (`>`) and "bitwise right shift" (`>>`).
// distinguish them while parsing, since they share a common token (`>`)
// also distinguish the different `>>` between bitwise right shift (`1 >> 3`) and
// generic function call (`f<T<U>>()`)
func defineGreaterThanOrBitwiseRightShiftExpression() {
```

Reply:
Great suggestion! Corrected and extended it further